tests/store/tracking/test_file_store.py

```diff
@@ -402,10 +408,14 @@ def test_create_experiment(self):
             exp1.artifact_location,
             path_to_local_file_uri(posixpath.join(self.test_root, created_id)),
         )
+        assert exp1.creation_time
```

Comment:
```suggestion
        assert exp1.creation_time is None
```

We should avoid assertion like this because it's unclear what is checked.